x/crisis/client/cli/tx.go

```diff
@@ -42,7 +47,7 @@ func NewMsgVerifyInvariantTxCmd(clientCtx client.Context) *cobra.Command {
 				return err
 			}
 
-			return tx.GenerateOrBroadcastTx(clientCtx, msg)
+			return tx.GenerateOrBroadcastTxCLI(clientCtx, cmd.Flags(), msg)
```

Comment:
This currently panic when broadcasting @aaronc: https://pastebin.com/vpnbXtQ4. Looks like it's using the default encoder.